system-addon/content-src/components/Dialog/Dialog.jsx

```diff
@@ -0,0 +1,84 @@
+const React = require("react");
+const {connect} = require("react-redux");
+const {FormattedMessage} = require("react-intl");
+const {actionTypes, actionCreators: ac} = require("common/Actions.jsm");
+
+/**
+ * Confirmation Dialog component.
+ * One primary action button, one cancel button.
+ *
+ * Content displayed is controlled by `data` prop the component receives.
+ * Example:
+ * data: {
+ *   // Any sort of data needed to be passed around by actions.
+ *   payload: site.url,
+ *   // Primary button SendToMain action.
+ *   action: "DELETE_HISTORY_URL",
+ *   // Primary button USerEvent action.
+ *   userEvent: "DELETE",
+ *   // Array of locale ids to display.
+ *   message_body: ["confirm_history_delete", "confirm_history_delete_notice"],
+ *   // Text for primary button.
+ *   confirm_btn_id: "menu_action_delete"
+ * },
+ */
+const Dialog = React.createClass({
```

Comment:
I think you should rename this to `ConfirmDialog` since it is specifically used for confirmations